app/test5.py

```diff
@@ -0,0 +1,20 @@
+def openai_api_call(prompt_text):
+    import openai
+    from os import getenv
+    from dotenv import load_dotenv
+
+    load_dotenv()
+
+    api_key = getenv('OPENAI_API_KEY')
+    if not api_key:
+        raise EnvironmentError("OPENAI_API_KEY is not set in the environment variables.")
```

Comment:
Raising a generic `EnvironmentError` might not be the best choice for missing environment variables as it's too broad. Consider creating a custom exception or using a more specific built-in exception to make error handling clearer and more specific.

**Label**: enhancement

**Code suggestion**

```python
if not api_key:
    raise ValueError("OPENAI_API_KEY is not set in the environment variables.")
```